### ISSUE TITLE
No license information generated?

### ISSUE THREAD

**Issue by garydgregory** [hexatomic/cff-maven-plugin#28]
Hello from Apache Commons,

I tried generating a CFF for https://github.com/apache/commons-io but what I got did not include the license field. Is this normal?

TY

**Comment by thomaskrause** [comment]
Yes, this is not implemented yet but could be similarly implemented as for the license information of external dependencies. We can't directly use the `license` field of the pom because [CFF requires SPDX identifiers](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#license), a requirement not found in the pom. Also, we would need to check if e.g. a `license-url` field already exists because we want to avoid having a mismatch between both these fields. 

As a workaround, you can add the license information once to the CITATION.cff file manually. When the `cff:create` goal is executed, all existing fields are kept as they are. Only non-existing fields are overwritten (especially the author list, since this information might change more frequently).
